src/frontend/src/Components/map/Map.js

```diff
@@ -410,23 +416,31 @@ export default function DriveBCMap(props) {
     // Do nothing if list empty
     if (filteredCameras) {
       // Deep clone and add group reference to each cam
-      const clonedCameras = JSON.parse(JSON.stringify(filteredCameras));
-      const finalCameras = addCameraGroups(clonedCameras);
+      const clonedCameras = JSON.parse(JSON.stringify(cameras));
```

Comment:
See https://developer.mozilla.org/en-US/docs/Web/API/structuredClone for modern alternative.